README.rst

```diff
@@ -23,11 +23,11 @@ Available Features
 ------------------
 
 * Displays ``Uptime``, ``Memory Status``, ``Load Status``, ``Addresses``, ``Neighbours`` of the **Device**
-* Displays ``Monitoring Graphs`` namely ``Uptime``, ``Packet Loss``, ``Round Trip Time``, ``Traffic`` in the form of lively charts
-* ``Monitoring Graphs`` can be viewed at resolutions of 1 day, 3 days, a week, a month and a year
+* Displays ``Charts`` namely ``Uptime``, ``Packet Loss``, ``Round Trip Time``, ``Traffic`` in the form of lively charts
+* ``Charts`` can be viewed at resolutions of 1 day, 3 days, a week, a month and a year
 * CSV Export of monitoring data
 * It allows you to add custom `Charts <https://github.com/openwisp/openwisp-monitoring/#openwisp_monitoring_charts>`_
-* It allows to create custom ``Metric``, select a ``Chart`` and set a ``Threshold``
+* It allows to create custom ``Metric``, select a ``Chart`` and set a ``AlertSettings``
```

Comment:
let's try to write documentation that is not only aimed at developers, so we don't necessarily have to refer to the model names, we should try to use more human readable names.